Extensions/AdMob/JsExtension.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
       .setName('AdMob Cordova plugin')
       .setDependencyType('cordova')
       .setExportName('admob-plus-cordova')
-      .setVersion('1.28.0')
+      .setVersion('2.0.0-alpha.18')
```

Comment:
as a general rule, let's avoid using alpha versions for something we're pushing live for everyone.
I'd got for the last stable known version